src/pages/en/guides/markdown-content.md

```diff
@@ -203,7 +203,7 @@ back to it later.
 
 ## Markdown Component
 
-Astro has a dedicated component used to let you render Markdown in `.astro` files. 
+> NOTE: The `<Markdown>` component does not work in SSR and may be removed before v1.0. It should should be avoided if possible. To use Markdown in your templates, use a seperate `.md` file and then `import` it into your template as a component. Read this [RFC Discussion](https://github.com/withastro/rfcs/discussions/179) to learn more.
```

Comment:
```suggestion
> NOTE: The `<Markdown>` component does not work in SSR and may be removed before v1.0. It should should be avoided if possible. To use Markdown in your templates, use a seperate `.md` file and then [`import` Markdown](/en/guides/markdown-content#importing-markdown) into your template as a component.
```
You could do something like this, now that #405 is merged.

Reply:
(Unless your intention really was to point to the Discussion/rationale, of course! But, I do think having the link to how to import Markdown content, now that we have it, works here.)

Reply:
Agreed! That was a bit of a placeholder